### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review
image

### CODE REVIEW

**Review by ashishkatiyar2404 — APPROVED** [review]
looks good to merge.
The alignment of 'remember me' could start from starting(it will look better if it comes just below the password) and 'forgot password' in the end

**Comment by BALAJI141031** [comment]
why we need search bar in login page?

File: common.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}
+.icon-ecomm > .material-icons
+{
+  font-size: 1.5rem;
+}
+:root 
+{   
+    --drop-hover-color: #ddd;    
+    --main-background-color: #fff;
+}
```
Comment (deekshithmd) [inline common.css]: code looks good to merge

**Review by deekshithmd — APPROVED** [review]

**Comment by ShrutiLakhani** [comment]
> why we need search bar in login page?

We usually see that on most websites

**Pull request merged**